src/main/java/com/quadfour/QuadFourController.java

```diff
@@ -34,5 +36,12 @@ public String readOne(Model model) {
     	model.addAttribute("taskDTO", taskDTO);
         return "index";
     }
+    
+    @RequestMapping("/searchResults")
+    public String searchResults(@RequestParam(value="searchTerm", required=true)String searchTerm)
+    {
+    String enhancedSearchTerm = searchTerm + "";
+    return "index";
```

Comment:
Should search results return to index?  It looks like this feature is not fully implemented.